.docker/Dockerfile.cli

```diff
@@ -23,3 +25,6 @@ RUN echo "memory_limit=-1" >> /usr/local/etc/php/conf.d/memory.ini \
     && rm -rf /usr/local/etc/php/conf.d/memory.ini
 
 COPY . /app
+
+RUN chgrp -R 0 /app/.keys && \
```

Comment:
I think this was added explicitly for budgets implementation of private content - I don't think we need this in the platform.

Reply:
Thanks @steveworley. I realised this when the build failed. I've removed and overridden in Budget.